src/uwtools/tests/drivers/test_forecast.py

```diff
@@ -3,18 +3,42 @@
 Tests for forecast driver.
 """
 
-
 from pathlib import Path
 from unittest.mock import patch
 
 import pytest
 from pytest import fixture, raises
-
-from uwtools import config
 from uwtools.drivers.driver import Driver
 from uwtools.drivers.forecast import FV3Forecast
 from uwtools.tests.support import compare_files, fixture_path
 
+from uwtools import config
+
+
+@fixture
+def slurm_props():
+    return {
+        "account": "account_name",
+        "nodes": 1,
+        "queue": "batch",
+        "scheduler": "slurm",
+        "tasks_per_node": 1,
+        "walltime": "00:01:00",
+    }
+
+
+def test_batch_script(slurm_props):
+    expected = """
+#SBATCH --account=account_name
+#SBATCH --nodes=1
+#SBATCH --ntasks-per-node=1
+#SBATCH --qos=batch
+#SBATCH --time=00:01:00
+""".strip()
```

Comment:
I'd hope to see that this batch script would have the mpirun command in it.

Reply:
At the moment, this is just checking the output of `scheduler.batch_script()` as used in `forecast()` and the mpirun command is handled by `run_cmd`... Actually now that you mention it, this brings up a question - `scheduler.batch_script()` just generates the batch system block, not the actual full .sh script with `!/bin/bash` and the run command. Is this something we should handle here in `forecast()`, or as part of `scheduler.batch_script()`?

Reply:
I'm having trouble finding one of the completed EX scripts as recheck reference in the AWS bucket that we have

Reply:
I think I'd opt to have `JobScheduler.batch_script()` include the initial `#!/bin/bash` line (because it will always be needed) and a trailing newline after the `#SBATCH` block; and then, in `FV3Forecast`, append the return value of `self.run_cmd()` to actually run FV3.